static/app/components/releases/releasesDrawer.tsx

```diff
@@ -78,6 +70,11 @@ export function ReleasesDrawer({
                   <DateTime date={start} /> <span>{t('to')}</span> <DateTime date={end} />
                 </Fragment>
               }
+              Visualization={chartRenderer?.({
+                releases,
+                start,
+                end,
+              })}
```

Comment:
I added this back, but removed passing the renderer from insightsTimeSeriesWidget